src/app/members/alert/vaccine-alert/vaccine-alert.page.ts

```diff
@@ -140,27 +149,34 @@ export class VaccineAlertPage implements OnInit {
         .catch((error: any) => console.error(error));
     }
   }
+
   async sendSMS(child: any) {
     const loading = await this.loadingController.create({
       message: "Loading"
     });
     await loading.present();
 
     for (let i = 0; i < child.length; i++) {
+      console.log("");
+      console.log("message No = " + (i + 1));
       let message = this.generateSMS(child[i]);
-      await this.sendAlertMsg(child[i].ChildId, child[i].Child.User.MobileNumber, message);
+
+      // await this.sendAlertMsg(child[i].ChildId, child[i].Child.User.MobileNumber, message);
```

Comment:
@basitshah94 
please remove your debug console logs and commit again

Reply:
done and pushed